code/Control/UserDefinedFormController.php

```diff
@@ -62,7 +63,8 @@ class UserDefinedFormController extends PageController
         'ping',
         'Form',
         'finished',
-        'confirmfolderform' => 'CMS_ACCESS_CMSMain',
+        'confirmfolderformschema' => 'CMS_ACCESS_CMSMain',
+        'ConfirmFolderForm' => 'CMS_ACCESS_CMSMain',
```

Comment:
I need the form to be accessible on it's own action because the Treeview needs an AJAX endpoint.